presentation/src/main/java/com/gta/presentation/ui/MainActivity.kt

```diff
@@ -65,4 +61,8 @@ class MainActivity : BaseActivity<ActivityMainBinding>(ActivityMainBinding::infl
     private fun hideBottomNav() {
         binding.bnvMain.visibility = View.GONE
     }
+
+    override fun onSupportNavigateUp(): Boolean {
+        return navController.navigateUp(appBarConfiguration) || super.onSupportNavigateUp()
+    }
```

Comment:
[공식문서](https://developer.android.com/guide/navigation/navigation-ui#action_bar)에 이렇게 나와있어서 한건데, 

`NavController.navigateUp` 는 `Returns: True if the NavController was able to navigate up.` 

`onSupportNavigateUp`는 `Returns: true if Up navigation completed successfully and this Activity was finished, false otherwise.`

라 되어있는거 보면 위는 Navigation 그래프 상에서 뒤로 가기 가능하면 true인 거 같고, 아래는 엑티비티 종료되면 true인거 같넹..?

Reply:
아 그럼 종료와 뒤로가기 모두 고려하는 그런 로직인가보구나 신기하네..!

Reply:
이 코드 너무 멋져요